Content.Client/Power/Substation/SubstationVisualsComponent.cs

```diff
@@ -0,0 +1,13 @@
+using Content.Shared.Power.Substation;
+
+namespace Content.Client.Power.Substation;
+
+[RegisterComponent]
+public sealed partial class SubstationVisualsComponent : Component
+{
+    [DataField("screenLayer")]
+    public string LayerMap { get; private set; } = string.Empty;
+
+    [DataField("integrityStates")]
```

Comment:
```suggestion
    [DataField]
```
DataField names now auto generate from the field name. You only need to specify it if you need the DataField name to be different from the class field name (which it usually should not be).